roles/geth/templates/genesis.json

```diff
@@ -1,20 +1,796 @@
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000e4": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000e5": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000e6": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000e7": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000e8": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000e9": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000ea": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000eb": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000ec": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000ed": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000ee": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000ef": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000f0": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000f1": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000f2": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000f3": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000f4": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000f5": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000f6": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000f7": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000f8": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000f9": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000fa": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000fb": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000fc": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000fd": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000fe": {
+            "balance": "0x1"
+        },
+        "00000000000000000000000000000000000000ff": {
+            "balance": "0x1"
+        },
+        "{{ alloc_address }}": {
+            "balance": "0x200000000000000000000000000000000000000000000000000000000000000"
+        }
     },
-    "coinbase": "0x0000000000000000000000000000000000000000",
```

Comment:
Sorry, could you revert the order here to match the original one? It will make the diff a bit easier to read.